test/CMakeLists.txt

```diff
@@ -68,7 +68,7 @@ function(add_test_executable TEST_TARGET OUT_VARIABLE)
     # link to roadrunner-static, the flags needed should be transitive
     #    add_compile_definitions(${TEST_TARGET} PRIVATE STATIC_RR)
 
-    add_dependencies(${TEST_TARGET} roadrunner-static gtest gtest_main gmock gmock_main rr-mockups)
```

Comment:
This breaks the python tests on windows, since the TestModelFactory and other *pyd's now need to link the roadrunner.dll which is located under the bin directory. We could add the roadrunner.dll to the site-packages/roadrunner/tests directory but then we should probably link the main roadrunner module dynamically as well (or have two copies of the roadrunner code). I'm going to link the static version again for now so I can continue developing some python tests and we can make this a separate issue if you think necessary.

Reply:
Ah, sorry.  Reverting it is fine.  The issue is just that with the increased size of the llvm binaries, these files are suddenly much larger than they were, and I was already at the edge of my hard drive size.  I'll move stuff around instead.